data/examples/declaration/value/function/pragmas-out.hs

```diff
@@ -1,4 +1,4 @@
-sccfoo = {-# SCC foo #-} 1
+sccfoo = {-# SCC "foo" #-} 1
```

Comment:
This is due to a change in an `Outputable` instance; we could choose to not use it and keep the previous behavior.

Reply:
I like the new behavior, it adds a little bit more of normalization :-)